tests/baselines/reference/restTuplesFromContextualTypes.types

```diff
@@ -265,8 +265,8 @@ f3((a, b, c) => {})
 f3((...x) => {})
 >f3((...x) => {}) : void
 >f3 : (cb: (x: number, args_0: boolean, ...args_1: string[]) => void) => void
->(...x) => {} : (x_0: number, x_1: boolean, ...x_2: string[]) => void
->x : [number, boolean, ...string[]]
+>(...x) => {} : (x: number, x_1: boolean, ...x_2: string[]) => void

```

Comment:
This change is a little funky - `x` isn;t consistent with `x_1`, but not a dealbreaker.

Reply:
`x` is just the name given in the original type, the rest are generated.